WMF Framework/Theme.swift

```diff
@@ -131,18 +132,382 @@ public extension UIColor {
+        tagText: .base100,
+        tagBackground: .accent70,
+        tagSelectedBackground: .accent50,
+        rankGradientStart: .accent70,
+        rankGradientEnd: .green50,
+        distanceBorder: .base70,
+        descriptionBackground: .accent70,
+        descriptionWarning: .yellow50,
+        inputAccessoryBackground: .darkBase10,
+        inputAccessoryButtonTint: .base90,
+        pageIndicator: .accent90,
+        pageIndicatorCurrent: .accent70,
+        unreadIndicator: .green50,
+        depthMarker: .base70,
+        diffTextAdd: .green50,
+        diffTextDelete: .red75,
+        diffHighlightAdd: nil,
+        diffHighlightDelete: nil,
+        diffStrikethroughColor: .red75,
+        diffContextItemBackground: .base10,
+        diffContextItemBorder: .base10,
+        diffMoveParagraphBackground: .darkBase20,
+        diffCompareAccent: .orange50,
+        diffCompareChangeHeading: .base0)
+    
+    fileprivate static let widgetLight = Colors(
+        baseBackground: .clear,
+        midBackground: .clear,
+        paperBackground: .clear,
+        chromeBackground: .clear,
+        popoverBackground: .clear,
+        subCellBackground: .clear,
+        overlayBackground: UIColor(white: 1.0, alpha: 0.4),
+        batchSelectionBackground: .accent90,
+        referenceHighlightBackground: .clear,
+        hintBackground: .clear,
+        hintWarningBackground: .clear,
+        animationBackground: .paleNavy,
+        overlayText: .base20,
+        searchFieldBackground: .lightSearchFieldBackground,
+        keyboardBarSearchFieldBackground: .base80,
+        primaryText: .base10,
+        secondaryText: .base10,
+        tertiaryText: .base20,
+        disabledText: .base30,
+        disabledLink: .base70,
+        chromeText: .base10,
+        link: .accent30,
+        accent: .green50,
+        border: UIColor(white: 0, alpha: 0.15),
+        shadow: .base80,
+        chromeShadow: .base80,
+        cardBackground: .base0,
+        selectedCardBackground: .base10,
+        cardBorder: .clear,
+        cardShadow: .base0,
+        cardButtonBackground: .base0,
+        secondaryAction: .accent30,
+        icon: nil,
+        iconBackground: nil,
+        destructive: .red50,
+        error: .red50,
+        warning: .yellow50,
+        unselected: .base50,
+        blurEffectStyle: .extraLight,
+        blurEffectBackground: .clear,
+        tagText: .clear,
+        tagBackground: .clear,
+        tagSelectedBackground: .clear,
+        rankGradientStart: .accent30,
+        rankGradientEnd: .green50,
+        distanceBorder: .base50,
+        descriptionBackground: .orange50,
+        descriptionWarning: .orange50,
+        inputAccessoryBackground: .base0,
+        inputAccessoryButtonTint: .base90,
+        pageIndicator: .accent90,
+        pageIndicatorCurrent: .accent30,
+        unreadIndicator: .green50,
+        depthMarker: .base20)
+    
+    fileprivate static let widgetDark = Colors(
```

Comment:
We no longer have diffText colors here or for widgetLight. Is that because they're not relevant to this one?

Reply:
Correct - they aren't relevant to the widgets. There are likely still other colors defined here that aren't relevant, but the history and diff colors were an easy block to remove for now.